opflexagent/test/test_endpoint_file_manager.py

```diff
@@ -563,3 +574,200 @@ def test_registered_endpoints(self):
             manager.declare_endpoint(port, mapping)
             self.assertEqual(set(['uuid2', port.vif_id]),
                              manager.get_registered_endpoints())
+
+
+class TestEndpointFileManagerDvs(TestEndpointFileManager):
```

Comment:
My goal was to have all of the same tests as the non-DVS use case, except have them be tested with non-compute EPs (the DVS version should behave the same for non-compute EPs). I was using this approach to try and minimize code duplication, and still have these tests. I am open to changing this if it makes more sense to have it inherit from TestEndpointFileManagerBase.